daemon/api.go

```diff
@@ -171,8 +171,8 @@ func sysInfo(c *Command, r *http.Request) Response {
 }
 
 type metarepo interface {
-	Details(string, string) ([]snappy.Part, error)
-	Find(string) ([]snappy.Part, error)
+	Details(string, string, string) ([]snappy.Part, error)
```

Comment:
Not a blocker, but these look much less cryptic with actual parameter names.. it'll look just a long:

IOW, (a, b, c string) instead of (string, string, string).